docs/community/index.md

```diff
@@ -24,11 +24,15 @@ this page][edit-this-page] and then [send us a PR][contributing-guidelines]!
 ## Blogposts
 
   - Andy Pearce’s series of blogposts on the features and architecture of
-    BlazingMQ: [BlazingMQ: Introduction][andy-pearce-introduction] and
-    [BlazingMQ: Clustering and Message Flow][andy-pearce-clustering-message-flow].
+    BlazingMQ: [BlazingMQ: Introduction][andy-pearce-introduction], [BlazingMQ:
+    Clustering and Message Flow][andy-pearce-clustering-message-flow],
+    [BlazingMQ: Hands ON][andy-pearce-hands-on], and [BlazingMQ: Client/Broker
+    Protocol][andy-pearce-client-broker].
```

Comment:
What about putting these blog posts one by line?

Reply:
Now that there are so many of them that's a good idea.